.env.example

```
@@ -0,0 +1,25 @@
+SEED_ADMIN_USERNAME=admin
+SEED_ADMIN_PASSWORD=ChangeMeNow123!
+DATABASE_SSL_REJECT_UNAUTHORIZED=true
+DATABASE_CA_CERT_PATH=/etc/ssl/certs/gimbal-ca.pem
+VPN_INTERFACE=wg0
+
+# Twilio Verify API (for OTP authentication via SMS, WhatsApp, Email)
+# Create a Verify Service in Twilio Console: https://console.twilio.com/us1/develop/verify/services
+TWILIO_ACCOUNT_SID=your-account-sid
+TWILIO_AUTH_TOKEN=your-auth-token
+TWILIO_VERIFY_SERVICE_SID=your-verify-service-sid
+TWILIO_PHONE_NUMBER=+1234567890
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The TWILIO_PHONE_NUMBER key should go before the TWILIO_VERIFY_SERVICE_SID key

(UnorderedKey)